src/Ducks/signUp.js

```diff
@@ -1,18 +1,16 @@
-// comments.js
-
 import axios from 'axios';
 import qs from 'qs';
-//import { browserHistory } from 'react-router';
 import { push } from 'react-router-redux';
-
+import { SubmissionError } from 'redux-form'
 // Constants
 import { API_URL } from '../API';
+
 // Actions
 // Define actions for each part of API request etc
-export const LOGIN_LOAD = 'LOGIN_LOAD';
-export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
-export const LOGIN_FAILURE = 'LOGIN_FAILURE';
-export const LOGIN_UPDATE = 'LOGIN_UPDATE';
+export const SINGUP_LOAD = 'SINGUP_LOAD';
```

Comment:
Spelling mistake SINGUP instead of SIGNUP

Reply:
I am on it.